cl/alerts/tasks.py

```diff
@@ -244,3 +309,76 @@ def send_unsubscription_confirmation(
     html = html_template.render(email_context)
     msg.attach_alternative(html, "text/html")
     msg.send()
+
+
+@app.task()
+def send_docket_to_webhook(
+    d_pk: int,
+    since: datetime,
+    webhook_recipients: list[str],
+) -> None:
+    """POSTS the DocketAlert to the recipients webhook(s)
+
+    :param d_pk: The Docket primary key
+    :param since: Start time for querying the docket entries
+    :webhook_recipients: The list of email recipients to send the webhook to
+    :return: None
+    """
+
+    docket_entries = DocketEntry.objects.filter(
+        date_created__gte=since, docket_id=d_pk
+    )
+    if docket_entries.count() == 0:
+        # No new docket entries.
+        return
+
+    webhooks = Webhook.objects.filter(
+        event_type=WebhookEventType.DOCKET_ALERT,
+        user__email__in=webhook_recipients,
+        enabled=True,
+    )
+
+    serialized_docket_entries = []
+    for de in docket_entries:
+        serialized_docket_entries.append(DocketEntrySerializer(de).data)
```

Comment:
I'm a bit surprised you can get away with such a simple serialization. Check out the hoops we have to go through for the bulk data serializer. https://github.com/freelawproject/courtlistener/blob/0c74a05a74188066f268c7023f2a12b0f583cb8e/cl/api/tasks.py#L160-L166

Reply:
I think I forget to comment on this:
Interesting. Yeah, I think using the `JSONRenderer()` we ensure the JSON we sent to Webhooks meets the same characteristics we have in API responses. So I changed to use `JSONRenderer()`. Before we were using the JSON encoding included on `requests` where you just needed to pass a `dict`.